integrations/jenkins/client.py

```diff
@@ -56,10 +91,13 @@ async def fetch_resources(
         while True:
             params = self._build_api_params(resource, page_size, page)
             base_url = self._build_base_url(parent_job)
+            logger.info(f"Fetching {resource} from {base_url} with params {params}")
 
             job_response = await self.client.get(f"{base_url}/api/json", params=params)
             job_response.raise_for_status()
-            jobs = job_response.json()["jobs"]
+            logger.info(f"Fetched {job_response.json()}")
```

Comment:
I see you're logging the entire response from the job, and this can make the logs hard to read. Can we use debug instead or remove it completely since I see you have another log in the next 2 lines?

Reply:
Oh, great catch. I actually meant to use debug